NSS-Day-Cohort-42/rare-API-media-moguls
# added action to check whether a subscription to an author exists for a user

#### Changes Made
1. Modified file `subscriptions.json` in `fixtures` to include a new subscription to author 3 for testing finding user's subs to a single author
1. Modified file `subscriptions.py` in `views` to include `get_single_current_subscription`. New action filters subs by current user, active (no ended_on), and then a single author.
​
#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout ap-subscriptions
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Nuke your whole situation
> Send a GET request to `http://localhost:8000/subscriptions/get_single_current_subscription` with a body `{ "author_id": 3 }`
> Do it again with `{ "author_id": 2 }`! Different subscription! Nice 
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
